runtime/onert/api/src/nnfw_api_internal.h

```diff
@@ -156,6 +157,8 @@ struct nnfw_session
   std::unique_ptr<onert::compiler::Compiler> _compiler;
   std::shared_ptr<onert::exec::Execution> _execution;
   std::shared_ptr<onert::frontend::custom::KernelRegistry> _kernel_registry;
+
+  std::shared_ptr<onert::util::TracingCtx> _tracing_ctx;
```

Comment:
Looks like `unique_ptr` is enough (raw pointers for the other references)

Reply:
fixed. thanks.